src/main.rs

```diff
@@ -306,38 +305,21 @@ fn find_ram_in_linker_script(linker_script: &str) -> Option<MemoryEntry> {
         line = eat!(line, ":");
         line = eat!(line, "ORIGIN");
         line = eat!(line, "=");
+        println!("Printing line: {}", line);
```

Comment:
Is this meant to be kept inside the actual output, or is it for debugging purposes?

_Same for the other `println!("Printing lineX ...")` below._

Reply:
Yup, same here, sorry about that!